components/automl/create_dataset_for_tables/component.py

```diff
@@ -0,0 +1,43 @@
+from typing import NamedTuple
+
+
+def automl_create_dataset_for_tables(
+    gcp_project_id: str,
+    gcp_region: str,
+    display_name: str,
+    description: str = '',
+    tables_dataset_metadata: dict = {},
+    retry=None, #=google.api_core.gapic_v1.method.DEFAULT,
+    timeout: float = None, #=google.api_core.gapic_v1.method.DEFAULT,
+    metadata: dict = None,
+) -> NamedTuple('Outputs', [('dataset_path', str), ('create_time', str), ('dataset_id', str)]):
+    '''automl_create_dataset_for_tables creates an empty Dataset for AutoML tables
+    '''
+    import sys
+    import subprocess
+    subprocess.run([sys.executable, "-m", "pip", "install", "google-cloud-automl==0.4.0", "--quiet", "--no-warn-script-location"], env={"PIP_DISABLE_PIP_VERSION_CHECK": "1"}, check=True)
```

Comment:
recommended  using ' as quote delimiter  to be consistent across.

Reply:
Done.